tasks/install-jenkins.yml

```diff
@@ -7,6 +7,8 @@
     dest: "{{ jenkins_war }}"
     # Sometimes the main Jenkins repository has network stability issues
     timeout: "{{ jenkins_download_timeout }}"
+    owner: "{{ jenkins_user }}"
```

Comment:
In 5f469f13d0b9662a1250580a1572c435001a68b2's commit message, "when run as unprivileged an user" reads weird.

Reply:
Yeah, typo-type-thing. Fixed in 8248209.